doc/make.texi

```diff
@@ -7626,9 +7626,33 @@ separators (@code{/}).  Note that, in contrast to @code{realpath}
 function, @code{abspath} does not resolve symlinks and does not require
 the file names to refer to an existing file or directory.  Use the
 @code{wildcard} function to test for existence.
+
+@item $(this_line)
+@findex this_line
+@cindex this_line, function
+The invocation @code{$(this_line)} expands to the current line
+number.  It is inspired by the @code{__LINE__} magical macro of the C
+programming language.
+
+
+@item $(this_file)
+@findex this_file
+@cindex this_file, function
+The invocation @code{$(this_file)} expands to the current filename,
+e.g. to @code{Makefile}.  It is inspired by the @code{__FILE__}
+magical macro of the C programming language.
+
+
+@item $(this_counter)
+@findex this_counter
+@cindex this_counter, function
+The invocation @code{$(this_counter)} expands to a unique, incremented,
+counter (in decimal). It is inspired by the @code{__COUNTER__} magical
+macro of the C programming language understood by GCC.
+
 @end table
 
-@node Conditional Functions, Foreach Function, File Name Functions, Functions
```

Comment:
I'm getting an in build because this `@node` line is missing: 

```$ make
make
Making all in lib
make[1]: Entering directory '/tmp/remake/lib'


make.texi:12429: @xref reference to nonexistent node `Conditional Functions'
make.texi:12435: @xref reference to nonexistent node `Conditional Functions'
make.texi:12442: @xref reference to nonexistent node `Conditional Functions'
make.texi:7030: @menu reference to nonexistent node `Conditional Functions'
make.texi:278: @detailmenu reference to nonexistent node `Conditional Functions'
make.texi:7455: Next reference to nonexistent `Conditional Functions'
make.texi:7708: Prev reference to nonexistent `Conditional Functions'
Makefile:918: *** [make.info] error 1

Reply:
Could you help (perhaps by patching the pull request). I am not familiar witrh `texinfo`. But you could use the C code and improve/correct the documentation. Thanks. [Basile Starynkevitch](http://starynkevitch.net/Basile/index_en.html
)

Reply:
Ok - #116 started.  (I thought when you open a PR there's a button that allows maintainers to add more commits. I didn't see that here so a new PR was started).

I don't see though that the `this_` functions do anything. See that PR for details.